packages/holocron/__tests__/holocronModule.spec.jsx

```diff
@@ -340,14 +365,28 @@ describe('holocronModule', () => {
     const reducer = () => fromJS({ x: 3 });
     const mergeProps = (stateProps, dispatchProps, ownProps) => {
       const xy = stateProps.moduleState.x * ownProps.y;
-      return Object.assign({}, stateProps, dispatchProps, ownProps, { xy });
+      return {
+        ...stateProps, ...dispatchProps, ...ownProps, xy,
```

Comment:
@10xLaCroixDrinker probably a #nitpick 😓but in your opinion would this be more readable on separate lines? i.e.:
```
return {
  ...stateProps,
  ...dispatchProps,
  ...ownProps, 
  xy,
};
```
the `xy` kinda sneaks in at the end on one line, might be more readily visible which portions are spread and which are not (added as direct properties)?

Reply:
trying this out with a code suggestion in https://github.com/americanexpress/holocron/pull/3/files#r371494636